lib/Auth/Utils.php

```diff
@@ -318,6 +319,30 @@ public static function processPIResponse($stateId, PIResponse $response)
             }
 
             $state['privacyidea:privacyidea']['transactionID'] = $response->transactionID;
+
+            // Search for the image
+            foreach ($response->multiChallenge as $challenge)
+            {
+                if (!empty($challenge->image))
+                {
+                    if (!empty($challenge->clientMode) && $challenge->clientMode === "interactive")
```

Comment:
This is why we should just pass the whole multi_challenge to the UI in the future.
(No changes required **now**, but i hope you see where this would be going if we keep adding stuff like this)